### ISSUE TITLE
connect pull requests to issues from the board using a 'connect' icon

### ISSUE THREAD

**Issue by ashumz** [waffleio/waffle.io#2354]
This would address 2 issues: 

1) Users often forget the correct language for connecting PR's to Issues 
2) Users aren't always aware that in Waffle, you can connect PR's to Issues without having to close them once the PR is merged into the default branch (not always ideal behavior for teams)

**Comment by beausmith** [comment]
This feature would be super helpful!

**Comment by mikaelarguedas** [comment]
+1
Having to go edit the fist comment of a PR to connect it to an issue is cumbersome when triaging waffle boards. That feature would be very helpful! :+1:
